index.js

```diff
@@ -398,8 +398,10 @@ InfluxDB.prototype._prepareValues = function (series) {
         if (timestamp) {
           if (timestamp instanceof Date) {
             line += ' ' + timestamp.getTime()
+          } else if (/^[0-9]+$/.test(timestamp)) {
+            line += ' ' + timestamp  // UNIX timestamp
           } else {
-            line += ' ' + timestamp
+            line += ' ' + new Date(timestamp).getTime()  // hopefully an RFC3339 string
```

Comment:
I'll go ahead and get this in for now. That's actually a previously existing bug in there with dates. Going to defer fixing that due to the upcoming 5.x release